### PR TITLE
Remove database

### PR DESCRIPTION
**Pull request by kevincarrogan**

### CODE REVIEW

**Comment by kevincarrogan** [comment]
This project was initially forked from the Trade Helpdesk project by the previous team and as such has come with a lot of additional baggage that isn't necessary for this project. The only thing that this is required to do is to provide a set of contact forms for users of the CHEG website.

This change is an initial change (there may be more) to streamline this project, mostly to remove anything redundant for future development ease and to lower the surface area for security issues.

In this particular case we are removing the need for a database and as such any code that would either have been using the database or only exists because of the existence of a database, this includes:

  - Django admin
    - Not required as it was only displaying database models
  - Django auth
    - Was only being used due to Django admin being present
  - SSO integration
    - Was only being used due to Django admin being present
  - Country app
    - This wasn't used and was just copied from the previous project
  - Users
    - Was only being used due to Django admin being present

File: Pipfile [hunk 1]
```diff
@@ -23,18 +23,15 @@ splinter = "==0.13.0"
 certifi = "==2018.11.29"
 chardet = "==3.0.4"
 decorator = "==4.3.0"
-dj-database-url = "==0.5.0"
 django-classy-tags = "==0.8.0"
 django-manager-utils = "==1.2.0"
-django-postgres-extra = "==1.21a12"
 django-query-builder = "==1.1.0"
```
Comment (simonwo) [inline Pipfile:28]: Is this still necessary if there are no databases?

File: config/settings/base.py [hunk 2]
```diff
@@ -34,20 +32,18 @@
...
 SECRET_KEY = env.str("DJANGO_SECRET_KEY")
 
-ADMIN_ENABLED = env.bool("ADMIN_ENABLED")
-
 # Application definition
 
 INSTALLED_APPS = [
-    "django.contrib.auth",
-    "django.contrib.admin",
     "django.contrib.contenttypes",
```
Comment (simonwo) [inline config/settings/base.py:47]: Hm, do you need this with no databases?

File: config/settings/base.py [hunk 3]
```diff
@@ -128,12 +126,7 @@
     {"NAME": "django.contrib.auth.password_validation.NumericPasswordValidator"},
```
Comment (simonwo) [inline config/settings/base.py:126]: Do you need these validators if there is no auth?

**Review by simonwo — APPROVED** [review]

File: config/settings/base.py (hunk 3 above)
Comment (kevincarrogan) [reply config/settings/base.py:126]: I'm planning to do another sweep of changes to get rid of more things after this. This one I wasn't completely sure whether I could get rid of or not but it's on my radar to get rid of in the future.

This is pretty much the same answer for the other comments as well.

**Pull request merged**